src/api/faultmanagement/FaultManagementAPI.js

```diff
@@ -25,43 +25,12 @@ export default class FaultManagementAPI {
         this.openmct = openmct;
     }
 
-    acknowledgeFault(fault, ackData) {
-        return this.faultActionProvider.acknowledgeFault(fault, ackData);
-    }
-
     addProvider(provider) {
         this.provider = provider;
     }
 
-    addFaultActionProvider(provider) {
-        this.faultActionProvider = provider;
-    }
-
-    getFaultModel(faultData, type) {
-        return {
-            type: type || faultData?.type,
-            fault: {
-                acknowledged: Boolean(faultData?.acknowledged),
-                currentValueInfo: {
-                    value: faultData?.parameterDetail?.currentValue?.engValue?.doubleValue,
-                    rangeCondition: faultData?.parameterDetail?.currentValue?.rangeCondition,
-                    monitoringResult: faultData?.parameterDetail?.currentValue?.monitoringResult
-                },
-                id: `id-${faultData?.id?.namespace}-${faultData?.id?.name}`,
-                name: faultData?.id?.name,
-                namespace: faultData?.id?.namespace,
-                seqNum: faultData?.seqNum,
-                severity: faultData?.severity,
-                shelved: Boolean(faultData?.shelveInfo),
-                shortDescription: faultData?.parameterDetail?.parameter?.shortDescription,
-                triggerTime: faultData?.triggerTime,
-                triggerValueInfo: {
-                    value: faultData?.parameterDetail?.triggerValue?.engValue?.doubleValue,
-                    rangeCondition: faultData?.parameterDetail?.triggerValue?.rangeCondition,
-                    monitoringResult: faultData?.parameterDetail?.triggerValue?.monitoringResult
-                }
-            }
-        };
+    hasActions() {
```

Comment:
Can we please name this `supportsActions()` for consistency with our other providers?